### ISSUE TITLE
pc_query backup

### ISSUE THREAD

**Issue by jonsnow1357** [geier/pycarddav#123]
HI,

When I try to save all the entries to a vcard file using:
pc_query -m -b file_name

the resulting file has the format:
BEGIN:VCARD
VERSION:3.0
...
END:VCARDBEGIN:VCARD
VERSION:3.0
...
END:VCARDBEGIN:VCARD
VERSION:3.0
...
END:VCARD

basically missing an '\n' between END:VCARD and the following BEGIN:VCARD.

Am I using ths utility wrong?

Thank you

**Comment by geier** [comment]
No, you probably hit a bug. If you want to do a backup, perhaps take a look at [vdirsyncer](https://github.com/pimutils/vdirsyncer), which is actively maintained.
